minions/minions.py

```diff
@@ -42,7 +43,210 @@ def chunk_by_section(
         sections.append(doc[start:end])
         start += max_chunk_size - overlap
     return sections
+    
```

Comment:
let's make a `utils/chunk.py` file and send all those func there. just import them here